gui/randogui.py

```diff
@@ -113,6 +116,35 @@ def __init__(self, options: Options):
         self.ui.enable_trick.clicked.connect(self.enable_trick)
         self.ui.disable_trick.clicked.connect(self.disable_trick)
 
+        # setup presets
+        self.default_presets = {}
+        self.user_presets = {}
+        self.ui.presets_list.addItem("[New Preset]")
+        sep_idx = 1
+        with open("gui/default_presets.json") as f:
```

Comment:
```suggestion
        with (RANDO_ROOT_PATH / 'gui' / 'default_presets.json').open() as f:
```

otherwise the exe breaks